pkg/lib/cockpit-components-dialog.jsx

```diff
@@ -285,7 +287,7 @@
     const closeCallback = function() {
         if (origCallback)
             origCallback.apply(this, arguments);
-        ReactDOM.unmountComponentAtNode(rootElement);
+        root.unmount();
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17939-20221121-134727-dbe868f6-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)